== TEST (2019-03-14 19:38:07) 
-- CODE:
print(180/4)
-- OUTPUT:
45.0
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-14 19:38:10) 
-- CODE:
print(180/4)
-- EXECUTION TIME:
0.109715
-- TEST CASE 1:
---- input:
xxx
---- correct output:
45.0
---- user output:
45.0
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-02 08:49:55) 
-- CODE:
print(180/4)
-- EXECUTION TIME:
0.107771
-- TEST CASE 1:
---- input:
xxx
---- correct output:
45.0
---- user output:
45.0
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-02 08:49:55) 
-- CODE:
print(180/4)
-- EXECUTION TIME:
0.072644
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1577/1329/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-02 10:16:32) 
-- CODE:
print(180/4)=
-- EXECUTION TIME:
0.136675
-- ERROR:
  File "/home/codebench/codes/1577/1329/main.py", line 1
    print(180/4)=
                ^
SyntaxError: invalid syntax

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-02 10:16:54) 
-- CODE:
print(180/4)
print(total=45)
-- EXECUTION TIME:
0.134526
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1577/1329/main.py", line 2, in <module>
    print(total=45)
TypeError: 'total' is an invalid keyword argument for this function

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-02 10:17:02) 
-- CODE:
print(45)
-- EXECUTION TIME:
0.125575
-- TEST CASE 1:
---- input:
xxx
---- correct output:
45.0
---- user output:
45
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
